spec/requests/merge_stories_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+describe 'merged stories', type: :request do
+  let(:user) { create(:user) }
+  let(:mod) { create(:user, :moderator) }
+  let(:merged_story) { create(:story) }
+  let(:story) { create(:story) }
+
+  context 'user votes on merged story' do
+    before {
```

Comment:
Multi line blocks tend to use the `do ... end` notation rather than the `{ ... }` notation.

Reply:
Thanks! Changed that before block